src/datumaro/components/merge/extractor_merger.py

```diff
@@ -58,8 +58,8 @@ def get(self, id: str, subset: Optional[str] = None):
                 if item is not None:
                     return item
 
-        s = self._subset[subset]
-        return s.get(id)
```

Comment:
When I added a test `test_can_convert_torch_framework_detection`, I got an error at https://github.com/openvinotoolkit/datumaro/blob/cf244a29199b9132f974e9f2d35bd497c555860c/src/datumaro/plugins/data_formats/coco/extractor_merger.py#L87 and this might be differ to the original `extractor_merger`'s one https://github.com/openvinotoolkit/datumaro/blob/cf244a29199b9132f974e9f2d35bd497c555860c/src/datumaro/components/merge/extractor_merger.py#L61. So I thought it was a type between "self._subsets" and "self._subset". On the other hands, most base class `_DatasetBase` has "self._subsets" https://github.com/openvinotoolkit/datumaro/blob/cf244a29199b9132f974e9f2d35bd497c555860c/src/datumaro/components/dataset_base.py#L232.

Reply:
Sorry I mistook it for Python's `dict.get()`. Resolve this conversation.